### ISSUE TITLE
Can we add a new label like "Will be closed"?

### ISSUE THREAD

**Issue by MarkusPiotrowski** [biopython/biopython#3305]
We have a number of open issues that are either somewhat off-topic or where the discussion has ended, e.g. because the OA never responded or didn't do what we asked him, or we are pretty sure that the problem is the OA's setup but not Biopython. 

Some examples: #2997, #2898, #2784, #2311

I'd like to suggest adding a label "Will be closed" (or "Ask for closing" or similar) to such issues and "ping" the OA. So he gets the chance to respond, and we can easily browse our issues and closed such labelled issues after a while (e.g. if the label is older than 4 weeks).

**Comment by peterjc** [comment]
Using a label does make sense for triage purposes. Hopefully there is a widely used convention we can adopt.

I've seen "needs more info", "Info Needed", and variants, but that is slightly different - perhaps better?

**Comment by MarkusPiotrowski** [comment]
Mmh, Github has a default "question | Indicates that an issue or pull request needs more information"

I have seen "inactive", which describes best what I mean, still it doesn't cover all cases. It should say "We regard this as closed and will close it anytime soon, unless you or someone else speaks up"

And anyone of us who sees this label can close this issue (if the label has a certain age), without having bad feelings. 

FYI, I found some nice articles about labeling, using prefixes seems to be a good idea to me:
https://medium.com/@dave_lunny/sane-github-labels-c5d2e6004b63
https://robinpowered.com/blog/best-practice-system-for-organizing-and-tagging-github-issues/
https://seantrane.com/posts/logical-colorful-github-labels-18230/

Sorry, I don't want to say that we should apply something similar, I just have never thought about labeling.

**Comment by chris-rands** [comment]
Do you think it's premature/impolite just to close these issues directly with a message and (optionally) a "Needs Info" style tag? The issues can always be re-opened if needed, and closed issues can still be can be viewed + commented on 

Another example #866

**Comment by peterjc** [comment]
I like "Needs info" as a short tag, and having an old issue tagged with that and closed is pretty clear (even without any text comment, which can be added too). e.g.

> We are closing this issue due to no activity, and more information being needed. Please feel free to re-open it with additional any information.

**Comment by MarkusPiotrowski** [comment]
Agree. Would be nice if we could add some explanation to the label, as in "Update" or "Good first issue"

**Comment by peterjc** [comment]
OK, would anyone else like to comment (paging @biopython/team-biopython), or shall we go with "Needs info" as a new label?
